src/client/datascience/interactive-window/nativeInteractiveWindow.ts

```diff
@@ -565,6 +655,30 @@ export class NativeInteractiveWindow implements IInteractiveWindowLoadable {
         }
     }
 
+    private combineData(
+        oldData: nbformat.ICodeCell | nbformat.IRawCell | nbformat.IMarkdownCell | undefined,
+        cell: ICell
+    ): ICell {
+        if (oldData) {
+            const result = {
+                ...cell,
+                data: {
+                    ...oldData,
+                    ...cell.data,
+                    metadata: {
+                        ...oldData.metadata,
+                        ...cell.data.metadata
+                    }
+                }
+            };
+            // Workaround the nyc compiler problem.
+            // eslint-disable-next-line @typescript-eslint/no-explicit-any
+            return (result as any) as ICell;
```

Comment:
yes scary when we cast to `any`, will only find out issues during run time
This shouldn't be a problem when we get rid of ICell

Reply:
I feel like we could us a no-explicit-any pass at some point, but not sure if we'd ever find the time. Looks like we have 350+ instances.